code/game/objects/effects/misc.dm

```diff
@@ -137,3 +137,19 @@
 	. = ..()
 	icon_state = "snap3"
 	render_target = "*snap[id]"
+
+/obj/effect/frosty_breath
+	icon = 'icons/effects/effects.dmi'
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	layer = ABOVE_MOB_LAYER
+	anchored = TRUE
+
+/obj/effect/frosty_breath/Initialize(mapload, mob/living/carbon/C)
+	. = ..()
+	dir = C.dir
+	if(C.buckled) //dont need to offset when they are upsidedown+buckled (on meat spike)
+		pixel_y = (C.buckled.buckle_offset + 16)
+	if(dir == NORTH)
+		layer = BELOW_MOB_LAYER
+	flick("breath_[istype(C.buckled, /obj/structure/kitchenspike) ? "180" : "[C.lying_prev]" ]", src) //you can thank the kitchenspike code not using lying_angle for this jank
```

Comment:
This is pretty nasty, I would prefer if you just made the kitchenspike set `lying_angle` instead.